scripts/ci/Dockerfile

```diff
@@ -65,40 +60,36 @@ RUN npm install -g \
 USER runner
 
 FROM output-base-with-pg AS output-express
-ARG GIT_SHA
-ENV DD_GIT_COMMIT_SHA="${GIT_SHA}"
-ENV DD_GIT_REPOSITORY_URL="github.com/island-is/island.is"
 
 COPY --from=builder /build/${APP_DIST_HOME} /webapp/
 
+ARG GIT_BRANCH GIT_SHA GIT_REPOSITORY_URL
+ENV GIT_BRANCH=${GIT_BRANCH} GIT_SHA=${GIT_SHA} GIT_REPOSITORY_URL=${GIT_REPOSITORY_URL}
+ENV DD_GIT_BRANCH=${GIT_BRANCH} DD_GIT_SHA=${GIT_SHA} DD_GIT_REPOSITORY_URL=${GIT_REPOSITORY_URL}
+LABEL branch=${GIT_BRANCH}
+LABEL commit=${GIT_SHA}
```

Comment:
But if I set `ARG`s early, we'll be cache-busting all later layers whenever they change, right? This would mean that if we use the git sha as an `ARG`, we cache-bust the docker layer caching on _every_ commit, right?

Reply:
Can we reduce duplication, and still avoid cache busts?